core/src/main/java/com/bumble/appyx/core/node/ParentNodeViewBase.kt

```diff
@@ -0,0 +1,9 @@
+package com.bumble.appyx.core.node
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.bumble.appyx.core.plugin.NodeAware
+
+interface ParentNodeViewBase<Routing : Any> : NodeView, NodeAware<ParentNode<Routing>> {
```

Comment:
While this would be a breaking change, I personally would prefer if we renamed the abstract class, and give this interface the non base name

Reply:
It would mean minimal changes to my current project so I have no opinion each way.
Please suggest the name for the implementation if it goes that way